### ISSUE TITLE
Cannot find module '../../node_modules/xcode'

### ISSUE THREAD

**Issue by elliotlings** [chemerisuk/cordova-plugin-firebase-hooks#1]
When running `cordova build ios` it is unable to find certain modules.

```
Error: Cannot find module '../../node_modules/xcode'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.Module._load (module.js:417:25)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Context.requireCordovaModule (/Users/elliot/Projects/myapp/myapp-app/node_modules/cordova-lib/src/hooks/Context.js:77:12)
    at module.exports (/Users/elliot/Projects/myapp/myapp-app/plugins/cordova-plugin-firebase-hooks/hooks/ios/copy-google-services.js:6:23)
    at runScriptViaModuleLoader (/Users/elliot/Projects/myapp/myapp-app/node_modules/cordova-lib/src/hooks/HooksRunner.js:187:18)
    at runScript (/Users/elliot/Projects/myapp/myapp-app/node_modules/cordova-lib/src/hooks/HooksRunner.js:165:16)
    at /Users/elliot/Projects/myapp/myapp-app/node_modules/cordova-lib/src/hooks/HooksRunner.js:133:20
    at _fulfilled (/Users/elliot/Projects/myapp/myapp-app/node_modules/q/q.js:787:54)
```

**Comment by chemerisuk** [comment]
@elliotlings hm, you could try `cd plugins/cordova-plugin-firebase-hooks && npm install`. Anyway version 1 of my firebase projects will use `<framework>` which now has cocopods support (will require cordova ios 4.3+)

**Comment by FdezRomero** [comment]
Unfortunately I'm having the same issue on Cordova iOS 4.3.0. I tried `npm install` from the plugin directory, but it didn't work.

I think npm 3.x+ installs the `xcode` dependency on `(project root)/node_modules` instead of `(project root)/plugins/cordova-plugin-firebase-hooks/node_modules` so it's not found by resolving `../../node_modules/xcode` as the error suggests.

If you give me some pointers maybe I can figure it out and submit a PR ;-)

**Comment by robations** [comment]
I managed to get past this point by hacking both `context.require...` lines in `copy-google-services.js` to use a standard `require("xcode")` require. I don't understand the code fully so I am not sure that this is a good fix.

**Comment by FdezRomero** [comment]
I manually fixed it undoing this commit: 089a5978076acf01b3cd599991a5219f79438271

```
xcode = context.requireCordovaModule('xcode'),
plist = context.requireCordovaModule('plist');
```

**Comment by ilyakar** [comment]
Can someone submit a pull request please?

**Comment by chemerisuk** [comment]
[Fixed](e2e360f3ae2e8cda2d14744babdec821b9e0e31e) in master
